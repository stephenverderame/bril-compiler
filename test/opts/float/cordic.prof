total_dyn_inst: 517
