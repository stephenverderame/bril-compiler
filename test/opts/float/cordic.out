0.86369602123419631
